mozilla-release/security/sandbox/win/src/sandboxbroker/sandboxBroker.cpp

```diff
@@ -452,6 +452,10 @@ SandboxBroker::SetSecurityLevelForContentProcess(int32_t aSandboxLevel,
     // Add rule to allow read access extensions directory within profile.
     AddCachedDirRule(mPolicy, sandbox::TargetPolicy::FILES_ALLOW_READONLY,
                      sProfileDir, NS_LITERAL_STRING("\\extensions\\*"));
+
+    // Add rule to allow read access features directory within profile.
+    AddCachedDirRule(mPolicy, sandbox::TargetPolicy::FILES_ALLOW_READONLY,
```

Comment:
@pauljt - I've got some discussing at mozilla irc at `#boxing` and `haik` suggested that you may be a good person to review changes in this fork